src/Raven.Server/Web/Studio/FolderPath.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Raven.Server.Config;
+using Sparrow.Platform;
+
+namespace Raven.Server.Web.Studio
+{
+    public static class FolderPath
+    {
+        private static readonly string FolderPathSuffix = PlatformDetails.RunningOnPosix ? "/" : "\\";
+
+        public static FolderPathOptions GetOptions(string path, bool isBackupFolder, RavenConfiguration ravenConfiguration)
```

Comment:
What is this for?

Reply:
This will display auto complete options when starting to write the path in the studio
![image](https://user-images.githubusercontent.com/6964706/50599226-18895180-0eb6-11e9-916c-a16b02e7c056.png)